### ISSUE TITLE
[NVDA] A11y_MicrosoftGuides_Outline_RearrangeSteps_Screenreader: NVDA doesn’t announce Name for the step controls.

### ISSUE THREAD

**Issue by ShwetaBaglane** [nvaccess/nvda#10056]
#### Logon Details as below:
Username: guidesselfhost@M365x699413.onmicrosoft.com
Instance: Test Test MRWNorthGuides Helf-host

### Steps to reproduce:
1. Open the Microsoft Dynamics 365 Guides (Preview) desktop application.
2. Sign-In into application using correct credentials.
3. Select the instance and click on Continue.
4. Navigate to any existing guides under the recent tab.
5. Start screen reader as NVDA.
6. Navigate to any step within any task using tab key or Press Up/ down arrow key.

### Actual behavior:
While NVDA is on, navigate to any step within any task using tab key. Screen reader doesn't announce Name for the step controls. Also if used up/ down arrow key, screen reader doesn't announce content of the step.

### Expected behaviour:
While NVDA is on, navigate to any step within any task using tab key. Screen reader should announce Name for the step control.

#### Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

#### User Impact:
Screen reader only users will face difficulty as they will not get to know the Name of the step.

#### MAS Reference:
MAS1.3.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

#### NVDA version:
2018.3.2

#### Windows version:
Win10 Enterprise 1903

**Comment by DrSooom** [comment]
@ShwetaBaglane: Same comment as in issue 10054:
> Please try to reproduce this with [NVDA 2019.2 RC2](https://ci.appveyor.com/api/buildjobs/qqofdhd6mlh8mq5u/artifacts/output/nvda_2019.2rc2.exe) (running via the Installer should be enough) and run NVDA before starting Microsoft Dynamics 365 Guides. Testing this with Narrator wouldn't be bad too. Thanks.

**Comment by ShwetaBaglane** [comment]
Tried to repro the issue as per the comment. But issue still reproduces on NVDA 2019.2 RC2 version.
Narrator announces correctly.

**Comment by feerrenrut** [comment]
@ShwetaBaglane The linked resources are not available to us, can you provide a minimal test case that demonstrates the issue in NVDA?

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
